scripts/src/setup.ts

```diff
@@ -0,0 +1,297 @@
+    );
+  }
+  if (process.env.GOOGLE_CLOUD_PROJECT !== config.common.googleCloud.project) {
+    logErrorAndExit(
+      "GOOGLE_CLOUD_PROJECT variable in .env does not match the project in config. Please ensure you have the correct values."
+    );
+  }
+
+  await confirm({
+    message:
+      "I've gotten the secret google-cloud-credentials.json file from a teammate and placed it in the config folder.",
+  });
+  let googleCloudCredentialsFile = "";
+  try {
+    googleCloudCredentialsFile = fs.readFileSync(
+      path.resolve(__dirname, "../../config/google-cloud-credentials.json"),
+      "utf8"
+    );
+  } catch (error) {
+    logErrorAndExit(
+      "google-cloud-credentials.json not found. Please ensure you have placed in the /config folder."
+    );
+  }
+  try {
+    const parsedGoogleCloudCredentialsFile = JSON.parse(googleCloudCredentialsFile);
+    if (parsedGoogleCloudCredentialsFile.project_id !== process.env.GOOGLE_CLOUD_PROJECT) {
+      logErrorAndExit(
+        "google-cloud-credentials.json has a mismatched project_id. Please ensure you have the correct file."
+      );
+    }
+    if (
+      parsedGoogleCloudCredentialsFile.client_email !== config.common.googleCloud.serviceAccount
+    ) {
+      logErrorAndExit(
+        "google-cloud-credentials.json has a mismatched client_email. Please ensure you have the correct file."
+      );
+    }
+  } catch (error) {
+    logErrorAndExit(
+      "google-cloud-credentials.json is not valid JSON. Please ensure you have the correct file."
+    );
+  }
+
+  const pathToGoogleCloudCredentials = path.resolve(
+    __dirname,
+    "../../config/google-cloud-credentials.json"
+  );
+  await confirm({
+    message: `I've filled in the GOOGLE_APPLICATION_CREDENTIALS variable in .env with the path to my google-cloud-credentials.json file (${pathToGoogleCloudCredentials}).`,
+  });
+  if (
+    path.resolve(__dirname, "../../config/google-cloud-credentials.json") !==
+    process.env.GOOGLE_APPLICATION_CREDENTIALS
+  ) {
+    logErrorAndExit(
+      `Please ensure you have set the GOOGLE_APPLICATION_CREDENTIALS variable in .env to the path of your google-cloud-credentials.json file (${pathToGoogleCloudCredentials}).`
+    );
+  }
+
+  console.log(chalk.green("\n✓ Config setup successful\n"));
+
+  await input({
+    message:
+      "Great! The next step is checking your database setup. Please ensure you have a local MongoDB server and Postgres server running locally. We use both of these database systems for different services since they have different strengths. \n\n Press enter to continue...\n",
+  });
+
+  console.log("Checking MongoDB database setup...");
+  const mongoClient = new MongoClient("mongodb://localhost");
+  try {
+    await mongoClient.connect();
+  } catch (error: any) {
+    console.log(
+      "Could not connect to MongoDB. Please ensure your connection string in .env has the correct user and password. " +
+        error.message
+    );
+  }
+
```

Comment:
can we log a successful mongodb connection message here before checking the postgres setup